stacker_blueprints/dynamodb.py

```diff
@@ -147,15 +153,14 @@ def create_scaling_iam_role(self):
             )
         )
 
-    def create_scalable_target_and_scaling_policy(self, asc, capacity_type="read"): # noqa
-        if capacity_type.lower() not in ("read", "write"):
+    def create_scalable_target_and_scaling_policy(self, table, asc, capacity_type="read"): # noqa
```

Comment:
This default feels obscure to me... is there a good reason for it? Seems like you'd always want to explicitely say if it's read or write, but I may be mistaken.